controllers/fluentbitconfig_controller.go

```diff
@@ -102,12 +103,13 @@ func (r *FluentBitConfigReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		if err != nil {
 			return ctrl.Result{}, err
 		}
-		if err = r.List(ctx, &parsers, client.InNamespace(req.Namespace), client.MatchingLabelsSelector{Selector: selector}); err != nil {
+		if err = r.List(ctx, &parsers, client.MatchingLabelsSelector{Selector: selector}); err != nil {
 			return ctrl.Result{}, err
 		}
 
+		ns := os.Getenv("NAMESPACE")
```

Comment:
It needs to support specifying the namespace where the secret is in. If not specified, the defaults namespace is the namespace of the operator.